packages/amplify-graphql-schema-generator/src/datasource-adapter/pg-datasource-adapter.ts

```diff
@@ -43,6 +43,16 @@ export class PostgresDataSourceAdapter extends DataSourceAdapter {
 
   private readonly PRIMARY_KEY_INDEX_NAME = 'PRIMARY';
 
+  private readonly FETCH_ENUMS_QUERY = `
+    SELECT  n.nspname AS enum_schema,
+      t.typname AS enum_name,
+      e.enumlabel AS enum_value
+    FROM    pg_type t JOIN
+      pg_enum e ON t.oid = e.enumtypid JOIN
+      pg_catalog.pg_namespace n ON n.oid = t.typnamespace
+    WHERE   n.nspname = 'public'
```

Comment:
There's some sort of weird whitespacing in here.

Reply:
This is formatted using SQL code style. Text after SELECT, FROM and WHERE clause are aligned.